parseutil/parseutil.go

```diff
@@ -106,19 +106,28 @@ func ParseDurationSecond(in interface{}) (time.Duration, error) {
 			return dur, nil
 		}
 		var err error
-		// Look for a suffix otherwise its a plain second value
-		if strings.HasSuffix(inp, "s") || strings.HasSuffix(inp, "m") || strings.HasSuffix(inp, "h") || strings.HasSuffix(inp, "ms") {
+
```

Comment:
Alternate approach to consider: try ParseInt first.  If that returns success, we don't have to worry about suffix.  If ParseInt fails, look for and handle a `d` suffix, failing which, use ParseDuration.

Reply:
I think you're approach is cleaner. 👍